lib/random.d.ts

```diff
@@ -37,25 +37,22 @@ declare module "@offirmo/random" {
 		static die(sideCount: number): (engine: Engine) => number;
 		static dice(sideCount: number, dieCount: number): (engine: Engine) => number;
 		static uuid4(engine: Engine): string;
-		static string(engine: Engine, length: number): string;
```

Comment:
I'm not sure what this is calling. `Random.string` looks like `(pool?) => (engine, length) => string`